src/releng_docs/projects/shipit-pulse-listener.rst

```diff
@@ -0,0 +1,27 @@
+.. shipit-pulse-listener-project:
+
+Project: shipit-pulse-listener
+==============================
+
+:contact: `Release Management`_
+
+This is a monitoring component that reacts to pulse messages, like:
```

Comment:
Please add that it's hosted on Heroku, as a worker dyno, and that we use several instances (production, staging, testing)